pkgs/servers/apache-flume/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, jre, makeWrapper, bash, coreutils, gnused, hadoop,
+  majorVersion ? "1.9.0" }:
+
+let
+  versionMap = {
+    "1.9.0" = {
+      flumeVersion = "1.9.0";
+      sha256 = "069nnakibv4pdra06fnq08nb7qypzx8j586r4fmw8kflpxdfswq3";
+    };
+  };
+in
+
+with versionMap.${majorVersion};
+
+stdenv.mkDerivation rec {
+  version = "${flumeVersion}";
+  name = "apache-flume-${version}";
+
+  src = fetchurl {
+    url = "mirror://apache/flume/${version}/apache-flume-${version}-bin.tar.gz";
+    inherit sha256;
+  };
+
+  buildInputs = [ hadoop jre makeWrapper bash gnused coreutils ];
```

Comment:
```bash``` ```gnused``` ```coreutils``` are already included in every bulid by default

Reply:
Fixed, thanks! Old habit's die hard (: